provider/aws/aws.go

```diff
@@ -660,6 +660,9 @@ func (p *AWSProvider) newChanges(action string, endpoints []*endpoint.Endpoint)
 	changes := make(Route53Changes, 0, len(endpoints))
 
 	for _, endpoint := range endpoints {
+		if endpoint == nil {
```

Comment:
If `endpoint == nil` there is no change to be made, so don't process any further.